processes/ingest/doab.py

```diff
@@ -30,145 +25,67 @@ class DOABProcess(CoreProcess):
     def __init__(self, *args):
         super(DOABProcess, self).__init__(*args[:4])
 
-        self.ingestOffset = int(args[5]) if args[5] else 0
-        self.ingestLimit = (int(args[4]) + self.ingestOffset) if args[4] else 10000
-
-        self.generateEngine()
-        self.createSession()
-
         self.s3_manager = S3Manager()
         self.s3_manager.createS3Client()
-        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.s3_bucket = os.environ['FILE_BUCKET']
 
-        self.fileQueue = os.environ['FILE_QUEUE']
-        self.fileRoute = os.environ['FILE_ROUTING_KEY']
+        self.file_queue = os.environ['FILE_QUEUE']
+        self.file_route = os.environ['FILE_ROUTING_KEY']
 
         self.rabbitmq_manager = RabbitMQManager()
         self.rabbitmq_manager.createRabbitConnection()
-        self.rabbitmq_manager.createOrConnectQueue(self.fileQueue, self.fileRoute)
+        self.rabbitmq_manager.createOrConnectQueue(
+            self.file_queue, self.file_route)
+
+        self.offset = int(args[5]) if args[5] else 0
+        self.limit = (int(args[4]) + self.offset) if args[4] else 10000
 
-        self.constants = get_constants()
+        self.dSpace_service = DSpaceService(base_url=self.DOAB_BASE_URL, source_mapping=DOABMapping)
```

Comment:
recommend doing d_space_service or dspace_service so that it's not a mix-match of camel and snake case.